src/progressbar/progressbar.js

```diff
@@ -54,6 +54,7 @@ angular.module('ui.bootstrap.progressbar', [])
         require: '^progress',
         scope: {
             value: '=',
+            max: '=?',
```

Comment:
I believe you wouldn't be able to set the max to be dynamic then. I thought using a variable from a different scope would be helpful here. 
Or did I understand you wrong?